benchmarks/benchmark/tools/latency-profile/manifest-templates/namespace.yaml.tpl

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
This duplicates the work that the ai-on-gke benchmark/infra does

ref: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/1f4c96861ad166fd8c2c7c1847e45c8b187e362c/benchmarks/infra/stage-2/main.tf#L22

this causes conflicts if the namespace already exists, which is likely if a user comes through the the infra/ pathway rather than the Bring Your Own Cluster pathway. I think we'll need a cleaner story to support both pathways

Reply:
See the readme references this as well, removed